auto_round_extension/qbits/qlinear_qbits.py

```diff
@@ -91,6 +95,20 @@ def __init__(
 
         self.trainable = trainable
 
+    def req_check(self):
+        torch_version = str(torch.__version__)
+        if QBITS_AVAILABLE:
+            itrex_version = str(intel_extension_for_transformers.__version__)
+            version_match_map = {"1.4": "2.2.0+cpu",
+                                "1.4.1": "2.2.0+cpu", "1.4.2": "2.3.0+cpu"}
```

Comment:
is it possible to automate it? otherwise, we always need to update it after release itrex

Reply:
let' make this version matching process automatic in next ITREX release